patches/node/feat_add_flags_for_low-level_hooks_and_exceptions.patch

```diff
@@ -52,26 +52,20 @@ index fee485f86e17f1987e140aea874656e5048cdbfe..7dda6b39c37cdf96f59a34ca6bc75ea2
  #include "node_internals.h"
  #include "node_main_instance.h"
  #include "node_metadata.h"
-@@ -128,6 +129,9 @@ using v8::Undefined;
+@@ -128,6 +129,6 @@ using v8::Undefined;
  using v8::V8;
  using v8::Value;
  
-+bool g_standalone_mode = true;
-+bool g_upstream_node_mode = true;
-+
  namespace per_process {
  
  // node_revert.h
-@@ -343,6 +347,13 @@ MaybeLocal<Value> Environment::RunBootstrapping() {
+@@ -343,6 +347,10 @@ MaybeLocal<Value> Environment::RunBootstrapping() {
  
    CHECK(!has_run_bootstrapping_code());
  
 +  if (g_standalone_mode) {
 +    isolate()->AddMessageListener(errors::PerIsolateMessageListener);
 +  }
-+  if (g_upstream_node_mode) {
-+    isolate()->SetFatalErrorHandler(OnFatalError);
```

Comment:
is this necessary ?

Reply:
nope! already deleted :)

Reply:
yes - it's already invoked in `SetIsolateUpForNode` so this needlessly does it twice.